PyPoE/poe/file/translations.py

```diff
@@ -1052,13 +1052,15 @@ def register_from_string(self, string: str):
                     try:
                         self.index_handlers[handler.id].append(int(args[0]))
                     except ValueError as e:
-                        warnings.warn('Broken quantifier "%s" - Error: %s' % (string, e.args[0]), TranslationWarning)
+                        warnings.warn('Broken quantifier "%s" - Error: %s' %
+                                      (string, e.args[0]), TranslationWarning)
```

Comment:
https://github.com/Project-Path-of-Exile-Wiki/PyPoE/pull/37#discussion_r744102703

Reply:
Fixing